include/mbgl/layermanager/location_indicator_layer_factory.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <mbgl/layermanager/layer_factory.hpp>
+
+namespace mbgl {
+
+class LocationIndicatorLayerFactory : public LayerFactory {
```

Comment:
nit: final class?

Reply:
It's made the same as all other factories, all methods are final